Automattic/wp-calypso
# Reader: Migrate Images to Webpack

#### Changes proposed in this Pull Request

As part of #36097, the images need to be migrated over to Webpack so the colour changes are reflected, but also the changes in #30255. I think the best thing to do is to do this per folder, so here's the Reader one.

#### Testing instructions

On an account that's never used the Reader before, verify that the banner still works the same. This is probably the least straightforward change in this PR:
<img width="969" alt="Screenshot 2019-10-15 at 20 39 19" src="https://user-images.githubusercontent.com/43215253/66865173-907cc000-ef8e-11e9-88b8-5054fb1fb3a3.png">

Verify that the App Block uses the new logo instead of the old one (I found it and migrated it from the `/wordpress` folder, it's not used anywhere currently. You can force it to appear by modifying this line to something that always returns true, like 2 > 1: 
https://github.com/Automattic/wp-calypso/blob/34f354c3440d8e4e86fd304756fa265ec3dabdae/client/reader/sidebar/promo.jsx#L28
<img width="266" alt="Screenshot 2019-10-15 at 19 59 05" src="https://user-images.githubusercontent.com/43215253/66865256-c8840300-ef8e-11e9-96a0-5f7f9807db76.png">

Verify the Conversations image is the same at `/conversations`
<img width="939" alt="Screenshot 2019-10-15 at 20 16 34" src="https://user-images.githubusercontent.com/43215253/66865258-c8840300-ef8e-11e9-90f8-b7e41cd83928.png">

Verify that the Reader image with a new account is the same, including with the Conversations empty content.
<img width="919" alt="Screenshot 2019-10-15 at 20 36 12" src="https://user-images.githubusercontent.com/43215253/66865260-c8840300-ef8e-11e9-9f38-585b2f24e314.png">

This same image existed twice for some reason, with no difference but some extra space. I think it's best to just use CSS to recreate that space, so I deleted one.
<img width="880" alt="Screenshot 2019-10-15 at 21 06 10" src="https://user-images.githubusercontent.com/43215253/66865682-a474f180-ef8f-11e9-8f2d-ff967857d79a.png">

Verify that `/start/reader/reader-landing` still has the same images

![fasdfsdafadsaf](https://user-images.githubusercontent.com/43215253/52582857-1952c300-2e26-11e9-83b1-c665fa238a5a.png)

Verify that the play icon is the same (I just searched "video" in the Reader, this was the first video that appeared)

<img width="252" alt="Screenshot 2019-10-16 at 20 15 16" src="https://user-images.githubusercontent.com/43215253/66951166-b5863700-f051-11e9-9995-85e29d72afd5.png">

Aaaand confirm that the images aren't used anywhere other than what's been updated in this PR. :)
